atomic_reactor/plugins/pre_inject_yum_repo.py

```diff
@@ -92,3 +92,5 @@ def run(self):
         yum_repos = list(self.workflow.files)
         add_yum_repos_to_dockerfile(yum_repos, df, inherited_user,
                                     self.workflow.builder.base_from_scratch)
+        for repo in yum_repos:
+            self.log.info("injected yum repo: %s", repo)
```

Comment:
Do we need multi-line log entry for this?

Reply:
I'm not sure, that's why I put it in a separate commit.

Reply:
More specifically, I wasn't sure if injected repos should be logged at all, and if there were logged, whether it was better to log them one per line or as a group.  If they need to be logged, I suggest 1 per line for easier grepping.